open_event/helpers/data.py

```diff
@@ -126,6 +128,24 @@ def create_session(form, event_id, is_accepted=True):
         update_version(event_id, False, "session_ver")
 
     @staticmethod
+    def add_session_to_event(form, event_id):
+        """
+        Session will be saved to database with proper Event id
+        :param form: view data form
+        :param event_id: Session belongs to Event by event id
+        """
+        new_session = Session(title=form["title"],
+                              subtitle="",
+                              description=form["description"],
+                              start_time=form["start_time"],
+                              end_time=form["end_time"],
+                              event_id=event_id,
+                              abstract="")
+
```

Comment:
@SaptakS i think that we need to assign speaker to session, We need also state of session like, pending, accepted, rejected etc.